pkg/secrets/clients/conjur/mocks/conjur_secrets_retriever.go

```diff
@@ -13,22 +13,44 @@ import (
 */
 var CanExecuteConjurVar bool
```

Comment:
Is this left in for backwards compatability?

Reply:
Thanks. Didn't spot this, removed!